R4ML/R/ml.model.svm.R

```diff
@@ -150,24 +151,24 @@ setMethod("initialize", "r4ml.svm",
 #' @examples \dontrun{
 #' 
 #' # Load the Iris dataset 
-#' iris_hf <- as.r4ml.frame(iris)
+#' iris_r4mlDF <- as.r4ml.frame(iris)
```

Comment:
I was thinking , r4mlDF kind of mix cases and also too long. I think  we can fix the doc bug and merge it in the IOP and then change _hf or _hm kind of code later.
what do you think?

or would it be easier to subsititute similar case and smaller name for r4mlDF with r4mldf or r4ml_df?